src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
+With applied intelligence's correlation logic, alerts and events from any source are automatically grouped into related issues.  As events come into your system they are instantly correlated based on time, alert context, and relationship data. If multiple events are related then our correlation logic will funnel the related incidents into a single, comprehensive [issue](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence#issue-feed). This ensures that your team is alerted to relevant issues as soon as possible while also reducing distracting and redundant alerts.
+
+We call this correlation logic **decisions**. We have built-in decisions but you can also create and customize your own on the decisions page **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Decisions**. The more you configure your decisions to best suit your needs,  the better New Relic can correlate your incidents, reduce noise and provide increased context for on-call teams.
 
 <img
-  title="NRAI_Decisions_Index.png"
-  alt="A screenshot that shows applied intelligence correlating incident data in the decisions UI."
-  src={nraiDecisionsIndex}
+  title="NRAI_Decisions_Page.png"
+  alt="A screenshot that shows applied intelligence decisions UI."
+  src={nraiDecisionsPage}
 />
 
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Applied intelligence > Incident intelligence > Decisions**: Our UI shows how each decision correlates incidents.
 </figcaption>
 
-## Key concepts [#key-concepts]
+### What is correlation and how does it work? [#what-is-correlaton]
 
-Here are some key concepts for understanding our decisions logic:
+Your most recent and active incidents are available for our correlation logic. For example, let's say your system has received two alerts saying a synthetic monitor is failing in Australia and London. These two alerts will have created their own, unique issues. Those issues will generate their own, unique incidents based on your teams existing [incident creation policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/#preference-target). New Relic's correlation logic will then test those incidents against each other to find similarities. In this case, it's the same monitor that is failing across multiple locations, so New Relic will merge both incidents into a single incident that contains each relevant event.
```

Comment:
erase comma between 'own, unique', that's done twice.

Reply:
done, thanks!